py/aou_workbench_client/concepts.py

```diff
@@ -62,6 +64,129 @@
     ('Vocabulary', 'vocabulary_id'),
     ('Count', 'count_value')]
 
+_CONCEPT_TABLE_HTML_TEMPLATE = """
+<script language="javascript">
+  var selected_row_id = null;
+  var selected_data = null;
+  
+  var old_selected_color = null;
+  
+  function select_concept(id, name, domain, vocabulary, standard) {
+    max_results = document.getElementById('max_results');
+    variable_prefix = document.getElementById('variable_prefix');
+    cohort_name = document.getElementById('cohort_name');
+    generate_code = document.getElementById('generate_code');
+    max_results.disabled = false;
+    max_results.style.color = '';
+    variable_prefix.disabled = false;
+    variable_prefix.style.color = '';
+    cohort_name.disabled = false;
+    cohort_name.style.color = '';
+    generate_code.disabled = false;
+    if (selected_row_id) {
+      document.getElementById('row_' + selected_row_id).style.backgroundColor = old_selected_color;
+    }
+    selected_row_id = id;
+    selected_data = { 'name': name, 'domain': domain, 'vocabulary': vocabulary, 'standard': standard };
+    new_selected_row = document.getElementById('row_' + id);
+    old_selected_color = new_selected_row.style.backgroundColor;
+    new_selected_row.style.backgroundColor = '#BBBBFF';    
+  }
+  
+  domain_to_table_map = {
+    'Condition': ['ConditionOccurrence', 'condition_concept_id', 'condition_source_concept_id'],
+    'Device': ['DeviceExposure', 'device_concept_id', 'device_source_concept_id'],
+    'Drug': ['DrugExposure', 'drug_concept_id', 'drug_source_concept_id'],
+    'Ethnicity': ['Person', 'ethnicity_concept_id', 'ethnicity_source_concept_id'],
+    'Gender': ['Person', 'gender_concept_id', 'gender_source_concept_id'],
+    'Measurement': ['Measurement', 'measurement_concept_id', 'measurement_source_concept_id'],
+    'Observation': ['Observation', 'observation_concept_id', 'observation_source_concept_id'],
+    'Procedure': ['ProcedureOccurrence', 'procedure_concept_id', 'procedure_source_concept_id'],
+    'Race': ['Person', 'race_concept_id', 'race_source_concept_id']
+  };
+  
+  function generate_python_code() {
+    max_results = document.getElementById('max_results').value;
+    prefix = document.getElementById('variable_prefix').value;
+    cohort_name = JSON.stringify(document.getElementById('cohort_name').value);
+    domain = selected_data['domain'];
+    table_data = domain_to_table_map[domain];
+    if (!table_data) {
+      return;
```

Comment:
Seems like we'd want some kind of error flow in these cases. What happens if you just throw a Javascript error here?

Reply:
Done.

Reply:
Out of curiosity - did you try this out in your notebook? Did the UI or notebook cell give any indication of the error?

Reply:
The UI doesn't give any indication. In the console, it says: 

VM73:46 Uncaught Unsupported domain: Meas/Procedure